hack/e2e/README.md

```diff
@@ -24,3 +24,24 @@ GINKGO_FOCUS=Dynamic.\*xfs.\*should.store.data \
 GINKGO_NODES=1 \
 ./hack/e2e/run.sh
 ```
+
+# git read-tree
+
+Reference: https://stackoverflow.com/questions/23937436/add-subdirectory-of-remote-repo-with-git-subtree
+
+How to consume this directory by read-treeing the ebs repo:
+
+```
+git remote add ebs git@github.com:kubernetes-sigs/aws-ebs-csi-driver.git
+git fetch ebs
+git read-tree --prefix=hack/e2e/ -u ebs/master:hack/e2e
+```
+
+To commit changes and submit them as a PR back to the ebs repo:
```

Comment:
Can you elaborate on this? Why are we doing this instead of a regular PR?

Reply:
if you've read-treed this director ythen youa re in another repo (efs). so if you want to commit your changes back to theoriginal repo (ebs) you need to do this. does it make sense, not sure what to add here

Reply:
Yeah got it. That's what I kind of guessed but wanted to make sure. lgtm.